user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+
+This document is meant as a guideline to follow in case your Travis CI Enterprise production installation becomes unresponsive or unavailable.
+
+If this happens, we'd like to ask you to follow the steps outlined in this document first before you get in touch with us to save important data and collect as much evidence as possible. This will help us to provide you the best support possible and focus completely on resolving the problem.
+
+## First: Don't panic
+
+The first thing you should do is: keep calm. We're here to help and we're confident that we can bring it back online together with you!
+
+## Prerequisites
+
+Please make sure that you have root access on all machines Travis CI Enterprise is running on. This is crucial because we will ask you to perform operations where you either need to be root or you need to have the appropriate `sudo` privileges. If you don't have direct root privileges, please make sure that a person who has is available.
+
+## Coordinating with Support
+
+We are here to help troubleshoot and fix any problems that may arise with your Travis CI Enterprise installation. For efficiency, though, we would request you minimize making changes to your setup that are not directly recommended by support, or document these steps and notify support accordingly.
+
+## Terminology
+
+To make sure that there are no confusions, we'd like to define a couple of terms:
+
+- `Platform machine`: The machine that runs most of the Travis web components. This is the machine your domain is pointing to.
+- `Worker machine`: The worker machine(s) run your builds.
+
+## Backup your encryption key
```

Comment:
"Back up"  when used as a verb

"very important building block " doesn't add much, so it becomes 

Without the encryption key you cannot access the information in your production database. To make sure that you can always recover your database, make a backup of this key:
Make this a note:

> Without this key the information in the database is not recoverable.